sql/updates/world/master/2024_xx_xx_xx_world.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `creature_static_flags_override` (  
```

Comment:
missing `DROP TABLE IF EXISTS` or use a procedure which checks if table exists before executing this statement